### PR TITLE
Workflow Automated Test - LRQA-67806 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

**Comment by PauloVAS** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase [hunk 1]
```diff
@@ -38,12 +38,61 @@ definition {
 	}
 
 	@description = "LPS-108138 - Verify that users can bulk-transition pending tasks assigned to themselves from more than 1 instance via Workflow Metrics"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanBulkTransitionPendingTasksFromMultipleInstances {
+		property portal.acceptance = "true";
```
Comment (rodrigocunhaa) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase:43]: You can remove this property

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase [hunk 2]
```diff
@@ -38,12 +38,61 @@ definition {
...
+
+		Blogs.addWithWorkflow(
+			entryContent = "Blogs Entry Content",
+			entrySubtitle = "Subtitle",
+			entryTitle = "Test Blogs 1");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Blogs Entry Content",
+			entrySubtitle = "Subtitle",
+			entryTitle = "Test Blogs 2");
```
Comment (rodrigocunhaa) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase:71]: Add the blogs through API. Do the following:

```suggestion
		JSONBlog.addEntry(
                      entryContent = "Blogs Entry Content",
                      entryTitle = "Blogs Entry Title 1");
                  
              JSONBlog.addEntry(
                      entryContent = "Blogs Entry Content",
                      entryTitle = "Blogs Entry Title 2");
```

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase [hunk 3]
```diff
@@ -38,12 +38,61 @@ definition {
...
+			workflowAssetType = "Blogs Entry",
+			workflowTask = "Review");
+
+		WorkflowMetrics.goToWorkflowAllItems(workflowProcessName = "Single Approver");
+
+		WorkflowMetrics.goToBulkTransitionSelectTasks();
+
+		WorkflowMetrics.selectAllTransitionTasks();
+
+		MenuItem.click(menuItem = "Approve");
```
Comment (rodrigocunhaa) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase:91]: Replace "Approve" with "Reprove"

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase [hunk 4]
```diff
@@ -38,12 +38,61 @@ definition {
...
 
-		// TODO LRQA-67806 CanBulkTransitionPendingTasksFromMultipleInstances pending implementation
+		Button.clickDone();
 
+		Alert.viewSuccessMessage();
```
Comment (rodrigocunhaa) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase:95]: Here you should assert that the Process Step column is now displaying "Update" for both tasks.

**Review by rodrigocunhaa — CHANGES_REQUESTED** [review]
Hi @PauloVAS, could you please make the changes that I requested?

Thanks!

**Comment by PauloVAS** [comment]
@rodrigocunhaa, done!

**Comment by rodrigocunhaa** [comment]
Just started reviewing :)

**Comment by rodrigocunhaa** [comment]
Hi @PauloVAS ,

You still missed one thing. You should assert that the Update status is displayed for both tasks. Since it is a small fixed, I made it here: https://github.com/rodrigocunhaa/liferay-portal/pull/617

Thanks!

**Pull request closed without merging**